Jenkinsfile

```diff
@@ -10,7 +10,16 @@ pipeline {
     stages {
         stage("Build") {
             steps {
-              sh 'mvn -B -Dmaven.test.failure.ignore clean install site'
+                script {
+                    infra.runMaven(["-Dmaven.test.failure.ignore", "clean", "install", "site"])
+                }
+            }
+        }
+        stage("Integration Tests") {
+            steps {
+                script {
+                    essentialsTest(baseDir: "src/test/it")
```

Comment:
This does not follow Maven naming conventions. `src/it/` is used by Failsafe plugin. `src/test/<langname>/` is used for unit tests in various languages. I think you want something like `src/essentials/`.